docs/guide/acquiring_data/database.rst

```diff
@@ -298,30 +298,34 @@ and add new data via :meth:`Database.download`. This means: If you are not
 sure whether the required data already exists in the database, use
 :meth:`Database.fetch` and you use a method to save bandwidth!
```

Comment:
Is this paragraph about the difference between `fetch` and `download` still correct?